### DISCUSSION TITLE
Add Vue Syntax

### DISCUSSION THREAD

**Discussion by angelhdzmultimedia** [Idered/chalk.ist#31] [Ideas]
Awesome tool! Props to the team. 💚🔥👏

If you allow me, I want to request a feature: Add Vue Syntax.

For now I use HTML instead.

Wishing you all lots of health, peace, and success.

**Comment by Idered** [comment]
Added

**Reply by angelhdzmultimedia** [reply]
Amazing work! Happy holidays and happy new year. Wishing the chalk.ist team lots of health, peace, and success.
